### PR TITLE
feat(suite-native): use enabled networks for msg system settings validation

### LINKED ISSUE #13470: Coin Enabling - message system compliance
Make sure that message system can correctly detect which coins are enabled to correctly show banner based on this.

### PR DESCRIPTION
**Pull request by vytick**
Only show coin related messages to users with enabled coin.

I tested this locally.

## Related Issue

Resolve #13470

### CODE REVIEW

**Review by matejkriz — CHANGES_REQUESTED** [review]
It doesn't seem to be working for me. Probably it's only because of missing actions in isAnyOfMessageSystemAffectingActions as mentioned in the other comment https://github.com/trezor/trezor-suite/pull/14037#discussion_r1738768077. When I restarted the app, it reflects the changes.
image
image
image

File: suite-native/message-system/src/messageSystemMiddleware.ts [hunk 1]
```diff
@@ -7,7 +7,8 @@ import {
     getValidMessages,
     selectMessageSystemConfig,
 } from '@suite-common/message-system';
-import { deviceActions, selectAccountsSymbols, selectDevice } from '@suite-common/wallet-core';
+import { deviceActions, selectDevice } from '@suite-common/wallet-core';
+import { selectEnabledDiscoveryNetworkSymbols } from '@suite-native/discovery';
 
 const isAnyOfMessageSystemAffectingActions = isAnyOf(
```
Comment (matejkriz) [inline suite-native/message-system/src/messageSystemMiddleware.ts:16]: This needs to be updated, it is now missing the actions changing enabled coins...

Comment (vytick) [reply suite-native/message-system/src/messageSystemMiddleware.ts:16]: Good point! I was testing it via changing the message jws and didnt think of this obvious way 🙈 
ca22975ef8


https://github.com/user-attachments/assets/4969b43e-4a96-489c-9c4c-a3a3e27589d4

**Review by matejkriz — APPROVED** [review]

**Pull request merged**